portality/models/v1/shared_structs.py

```diff
@@ -5,7 +5,8 @@
     "structs" : {
         "bibjson" : {
             "fields" : {
-                "title" : {"coerce" : "unicode"},
+                "title" : {"coerce" : "unicode", "additional_fields":{"analyzer": "ascii_folded",
+                                                                                 "search_analyzer": "ascii_folded"}},
```

Comment:
is it possible to define a new coerce such as `unicode_ascii_folded` in `DATAOBJ_TO_MAPPING_DEFAULTS` rather than add new features `additional_fields`  ?

Reply:
Good thought. I will look into this

Reply:
resolved in latest refactor